templates/projects/js/amp_store.js

```diff
@@ -0,0 +1,18 @@
+<script type="text/javascript">
+$(document).ready(function () {
+  $('#input_data_class').change(function () {
+      if ($('#input_data_class').val().includes('L1')){
```

Comment:
Consider adding "L1" to a variable and referencing it in the two conditional statements.

```javascript
var dataClassification = 'L1';

if ($(e.target).val().includes(dataClassification)) { ... }
```